torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -1129,10 +1129,6 @@ def call_function(
             tracer=tracer,
         )
         post_guards = tx.output.guards
-        if body_lifted_freevars:
-            for freevar in body_lifted_freevars.keys():
-                if "saved_tensor_marked" not in freevar.node.meta:
-                    unimplemented("NYI - freevars in autograd function.")
```

Comment:
We still need this check: our current strategy for autograd.Function can't handle freevars. It seems like we don't have a test for this, could you add one?